particles/tlv/particle.py

```diff
@@ -7,9 +7,12 @@
 import math
 
 class Particle(BaseParticle, RootObj):
-    def __init__(self, pdgid, charge, tlv, status=1):
+    def __init__(self, pdgid, charge, tlv, status=1, index=None, subtype='u'):
```

Comment:
Do not provide the index, get it from rootobj in all cases (_objid)

Reply:
Done